corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -53,6 +53,9 @@ <h4 class="modal-title">
                     <p class="help-block" data-bind="visible: appearance() == 'daterange'">
                       {% trans 'In "YYYY-MM-DD to YYYY-MM-DD" format' %}
                     </p>
+                    <p class="help-block" data-bind="visible: isMultiselect">
+                      {% trans 'e.g. Use #,# as seperator to select multiple values' %}
```

Comment:
@snopoke I actually ended up reverting the change you suggested as that's not necessary. Users will have to use `#,#` as separator, because the default values will be specified as list of strings that may contain commas or spaces. (they don't use indices)